projects/pionex-cex/index.js

```diff
@@ -0,0 +1,53 @@
+const { cexExports } = require('../helper/cex')
+
+const config = {
+  ethereum: {
+    owners: [
+      "0xf6d4e5a7c5215f91f59a95065190cca24bf64554",
+      "0x2a8a276019d3ec549ae657c945ef60aab4e33c9b",
```

Comment:
why add a bunch of empty wallets?

Reply:
We have multiple backup wallet addresses and plan to transfer assets to these addresses in the following weeks. If empty wallets cause trouble, we can also delete these wallet addresses and resubmit them after the asset transfer. Do we need to do this?

Reply:
@fightingm thanks for the explanation, no, it is fine.